generator/src/main/java/org/corfudb/generator/Correctness.java

```diff
@@ -9,10 +9,6 @@
  */
 public class Correctness {
 
```

Comment:
moving the constructor so the static members are the first statements in the class